data-prepper-plugins/s3-source/src/test/java/com/amazon/dataprepper/plugins/source/SqsWorkerTest.java

```diff
@@ -72,6 +77,8 @@ void processSqsMessages_should_return_number_of_messages_processed() {
         final int messagesProcessed = sqsWorker.processSqsMessages();
 
         assertThat(messagesProcessed, equalTo(1));
+        verify(s3Service, times(1)).addS3Object(any(S3ObjectReference.class));
```

Comment:
You don't need `times(1)` - this is the default behavior of Mockito. But, if you prefer having it there to be clearer, I'm fine to retain it.